src/localkiwixserver.cpp

```diff
@@ -43,6 +43,8 @@ LocalKiwixServer::LocalKiwixServer(QWidget *parent) :
     for (const auto &interfacePair : interfacesMap) {
         QString ip = QString::fromStdString(interfacePair.second.addr);
         interfaces.push_back(ip);
```

Comment:
Won't an `if (!ip.isEmpty())` check be justified here?

Reply:
Would not hurt to add for sure. I will go ahead and do it.